sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/PreferenceTokenStorage.java

```diff
@@ -0,0 +1,53 @@
+package com.microsoft.appcenter.utils.context;
+
+import android.content.Context;
+
+import com.microsoft.appcenter.utils.crypto.CryptoUtils;
+import com.microsoft.appcenter.utils.storage.SharedPreferencesManager;
+
+/**
+ * Storage for tokens that uses {@link SharedPreferencesManager}. Handles saving and encryption.
+ */
+public class PreferenceTokenStorage implements ITokenStorage {
```

Comment:
Can we call this AuthTokenStorage? I think it's a bit more clear and this is what we decided to call it in the Apple SDK